lib/omniauth/strategies/developer_admin.rb

```diff
@@ -0,0 +1,45 @@
+module OmniAuth
+  module Strategies
+    # The DeveloperAdmin strategy is for use in development only.
```

Comment:
Can we get a paragraph added to the README about this?

Reply:
Yes! What would be most helpful to you as an onboarder? Looking at what's in there right now, there's a reference to copying the `.env.development.example` to `.env.development`, which means that there will be defaults filled in for you when the sign-in form starts, so all you need do is click "Sign in" in development. Do we just need to state that or is there something else you're looking for?

Reply:
(and the only reason `ADMIN_EMAILS` will be there by default is because you committed it to the `.example` today, thanks @lozette!)

Reply:
Added a para in README about `/admin` and login with `DeveloperAdmin`